src/index.html

```diff
@@ -110,90 +110,88 @@
-                            </ol>
-                        </div>
+                        <div class="header">Is it secure?</div>
+                        <div class="description">We put a lot of effort into making it as secure as we could. Your personal data never leaves your phone. Apple requires a digital signature to open a pass. To generate this, the app only sends a checksum of your data to our server, and it's impossible to guess the original content. The last step happens on your phone: you preview your information before saving the pass. We don't track anyone, we don't collect any personal information, and we don't use cookies.</div>
                     </div>
                 </div>
                 <div class="item">
                     <i class="info icon"></i>
                     <div class="content">
-                        <div class="header">How can I get my EU DCC (certificate) ?</div>
-                        <div class="description">
-                            <p>In &#x1F1EB;&#x1F1F7; France, you can download it from <a href="https://attestation-vaccin.ameli.fr/" title="Visiter le site Ameli.fr pour télécharger votre certificat">Ameli's dedicated website</a></p>
-                            <p>In &#x1F1F1;&#x1F1FA; Luxembourg, you can download it from <a href="http://guichet.lu/certificat-vaccination">myGuichet</a></p>
-                            <p>In &#x1F1E8;&#x1F1ED; Switzerland, the certificates are distributed by your canton. Get more information on <a href="https://www.bag.admin.ch/bag/fr/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/covid-zertifikat.html">Federal Office of Public Health</a>.</p>
-                        </div>
+                        <div class="header">Can I see the code?</div>
+                        <div class="description">Of course. You can read the source code and dig more into how it works on <a href="https://github.com/clawfire/covid19-passbook-generator" title="open the project's Github page">the Github page of the project</a>. You can even contribute if you want!</div>
                     </div>
                 </div>
                 <div class="item">
                     <i class="info icon"></i>
                     <div class="content">
-                        <div class="header">How to save the certificate on Android?</div>
+                        <div class="header">Can I save the certificate on Android?</div>
                         <div class="description">
-                            <p>On Android, you can save the .pkpass file and open it with an application that supports this format. Here are some:</p>
+                            <p>You can save the .pkpass file and open it with an application that supports this format, for example:</p>
```

Comment:
Autant y aller fort et dire "Yes you can ! Save the ..." 😁  #yeswecan

Reply:
:-)